script.rb

```diff
@@ -419688,7 +419688,7 @@ def raw_apply(snr)
 s.ins 0xc3, 9, 8, byte(0)
 s.ins 0x4f, :addr_0x926e2, [22, 0]
 s.ins 0x86, ushort(1003), byte(1), byte(1), s.layout('@r@v28/BER_00350.Indeed, @v28/BER_00360.if we consider it that way, this witch named BEATRICE certainly is interesting.')
-s.ins 0x86, ushort(1004), byte(1), byte(1), s.layout('@r@v28/BER_00370.She opened up a world @v28/BER_00380.that neither I, you, or Lambda knew about.')
+s.ins 0x86, ushort(1004), byte(1), byte(1), s.layout('@r@v28/BER_00370.She opened up a world that neither I, you, or Lambda knew about.@v28/BER_00380.')
```

Comment:
From my testing the pause feels fine here. I'm okay with voicewaits in the middle of sentences, generally, just not actual clickwaits or page breaks. Please revert this and don't make similar changes unless the pause is completely inappropriate.